tox.ini

```diff
@@ -60,6 +60,7 @@ ignore =
     S404 # Consider possible security implications associated with the subprocess module.
     S108 # Probable insecure usage of temp file/directory.
     S307 # Use of possibly insecure function - consider using safer ast.literal_eval.
+    S602 # subprocess call with shell=True identified, security issue.
```

Comment:
Do you really want to deactivate the security warning everywhere?

Reply:
I'm planning to take this off once I get gh actions to pass and address this later.